src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -127,6 +127,20 @@ def __init__(self, parent=None, value=0.0):
         self.setLayout(qt.QHBoxLayout())
         self.layout().setContentsMargins(0, 0, 0, 0)
         self._numVal = FloatEdit(parent=self, value=value)
+
+        self._iconAuto = icons.getQIcon('scale-auto')
+        self._iconFixed = icons.getQIcon('scale-fixed')
```

Comment:
Maybe have the `scale-fixed` with something like: ~~AUTO~~

Reply:
I have removed the orange color for better contrast, and used a strocked auto for the fixed.
It's much better.